### ISSUE TITLE
The back button does not have a text on mouse over

### ISSUE THREAD

**Issue by RonnyB71** [Altinn/app-frontend-react#3123]
### Description

The arrow button in the upper left corner above the form does not have a text representation.

![Image](https://github.com/user-attachments/assets/e21ba8f5-f0ea-4784-8797-cd56f73dad67)

**Comment by bjosttveit** [comment]
This button was deleted in #2831

**Comment by olemartinorg** [comment]
Closing this as no longer relevant
